mne/epochs.py

```diff
@@ -2560,6 +2562,136 @@ def compute_psd(
             **method_kw,
         )
 
+    @verbose
+    def compute_tfr(
+        self,
+        method,
+        freqs,
+        *,
+        tmin=None,
+        tmax=None,
+        picks=None,
+        proj=False,
+        output="power",
+        average="auto",
+        return_itc=False,
+        decim=1,
+        n_jobs=None,
+        verbose=None,
+        **method_kw,
+    ):
+        """Compute a time-frequency representation of epoched data.
+
+        Parameters
+        ----------
+        %(method_tfr_epochs)s
+        %(freqs_tfr)s
+        %(tmin_tmax_psd)s
+        %(picks_good_data_noref)s
+        %(proj_psd)s
+        %(output_compute_tfr)s
+        average : bool | "auto"
+            Whether to return average power across epochs (instead of single-trial
+            power). Default is "auto" which means ``True`` if method="stockwell" and
+            ``False`` otherwise. ``average=True`` is not compatible with
+            ``output="complex"`` or ``output="phase"``.
```

Comment:
Seems like it would be nicer to have the default just be `True` or `False` so people could switch methods without the output type changing

Reply:
done in a303b38